docs/using-hive.md

```diff
@@ -1251,6 +1253,8 @@ spec:
     name: pull-secret
 ```
 
+Note: For `metadataJSONSecretRef`, copy its correspoding secret when the cluster is already managed by Hive and it appears in the original ClusterDeployment YAML;Otherwise create the metadata.json following the below samples.
```

Comment:
```suggestion
Note: For `metadataJSONSecretRef`:
- If the referenced Secret is available -- e.g. if the cluster was previously managed by hive -- simply copy it in.
- If you have the original metadata.json file -- e.g. if the cluster was provisioned directly via openshift-install -- create the Secret from it using the procedure described below.
- Otherwise, you may need to compose the file by hand. See the samples below.
```